cmd/galaxy_install.go

```diff
@@ -25,7 +25,7 @@ func (c *GalaxyInstallCommand) Run(args []string) int {
 		return 1
 	}
 
-	galaxyInstall := execCommand("ansible-galaxy", "install", "-r", "requirements.yml")
+	galaxyInstall := execCommand("ansible-galaxy", "install", "-r", "galaxy.yml")
```

Comment:
@swalkinshaw  @knowler 

What do you think if we check for `galaxy.yml` existence and fallback to `requirements.yml` for older versions of Trellis?

Reply:
yep that's what in my other comment 👍